src/node_bindings/era_test_node.rs

```diff
@@ -457,6 +456,22 @@ mod tests {
     //     let _ = EraTestNode::new().block_time_f64(0.5).spawn();
     // }
 
+    #[tokio::test(flavor = "multi_thread")]
+    async fn fork_initializes_correct_chain_id() {
+        let chain_id = 92;
+        let era_test_node = EraTestNode::new().chain_id(chain_id).spawn();
+        let rpc_url = era_test_node.endpoint_url();
+        let provider = ProviderBuilder::new()
+            .with_recommended_fillers()
+            .on_http(rpc_url);
+
+        let returned_chain_id = provider.get_chain_id().await.unwrap();
+
+        assert_eq!(returned_chain_id, chain_id);
+
+        drop(era_test_node);
```

Comment:
This is generally not required, since the node will be dropped anyways once the scope ends.